### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

To simplify future PRs.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [x] I made spot refactorings.
* [ ] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by w84thesun — APPROVED** [review]
👍

File: .codecov.yml [hunk 1]
```diff
@@ -24,6 +24,11 @@ coverage:
 ignore:
   - cmd
 
+# https://docs.codecov.com/docs/flags#recommended-automatic-flag-management
+flag_management:
+  default_rules:
+    carryforward: false
+
```
Comment (noisersup) [inline .codecov.yml:31]: Can we close https://github.com/FerretDB/FerretDB/issues/2106 then?

**Review by w84thesun — APPROVED** [review]
:shipit:

**Review by noisersup — APPROVED** [review]

File: .codecov.yml (hunk 1 above)
Comment (AlekSi) [reply .codecov.yml:31]: I don't know yet. I briefly checked codecov.io documentation, and could no longer find the feature I was talking about… I will take another look after we have a few builds with this new config.

**Pull request merged**